admin/tests/integration/components/certification-centers/information_test.js

```diff
@@ -59,7 +57,7 @@ module('Integration | Component | certification-centers/information', function (
     this.set('isEditMode', false);
 
     // when
-    await render(
+    await renderScreen(
       hbs`<CertificationCenters::Information @certificationCenter={{this.certificationCenter}} @isEditMode={{this.isEditMode}} />`
     );
     await clickByLabel('Editer');
```

Comment:
Toujours plus de testing-library, je :heart:

Mais en fait l'intérêt de la testing-library est de remplacer nos helpers maisons `clickByLabel` et `fillInByLabel` (qui imitaient la testing-library et qui sont dépréciés) par la "vraie" implémentation. Le rôle de la testing-library est uniquement de fournir des helpers pour faire des queries sur le DOM et récupérer des éléments.

Par contre, on utilise toujours les tests helpers d'Ember (par ex `click` et `fillIn`) pour faire des actions sur ces éléments, ce n'est pas le rôle de la testing-library.

Utiliser `renderScreen` seul ça n'apporte rien, l'intérêt est que cette méthode retourne un objet `screen` qui a [les méthodes de la testing-library](https://testing-library.com/docs/queries/about#priority). Il y en a plein mais la recommandation est d'utiliser `getByRole` dès que c'est possible.

```suggestion
    const screen = await renderScreen(
      hbs`<CertificationCenters::Information @certificationCenter={{this.certificationCenter}} @isEditMode={{this.isEditMode}} />`
    );
    await click(screen.getByRole({ role: 'button', { name: 'Editer' });
```

Reply:
Merci, je n'avais en effet pas cette syntaxe en tête et je pensais qu'utiliser clickByLabel nous faisait gagner en lisibilité tout arrêtant d'utiliser les ember test helpers, ce que je pensais être un objectif